### DISCUSSION TITLE
How exactly do you get this working?

### DISCUSSION THREAD

**Discussion by Kushys** [AdGoBye/AdGoBye#87] [Q&A] [answered]
Finally pulled the trigger and downloaded this, got the latest 4.0.

When I open the exe it opens a cmd window for a split second before closing, I assume that's intended? I can't see a background process for this though so maybe it isn't. I'm not sure if I should run it from a specific location or if I need to run it every time I start the game, etc. 
Either way it seems to not be working for me. I tested on LS Media and still saw all the ads every time I tried running it.

I feel a bit dumb seeing as it's just a single exe and I'm generally good with this kind of thing. Am I missing something or is something not working as intended?

**Comment by DubyaDude** [comment]
It closing is not normal, likely it crashed and the console didn't stay open.
Open a command prompt in that directory and run the executable in it so it doesn't close. If you're not sure how to do that please say so and I'll describe in greater detail.

**Comment by Kushys** [comment, marked as answer]
That revealed the issue. I needed a newer .net version, I don't have 8 installed. I can see now that it was mentioned in the 2.0 release of the program that you guys are using .net 8.
Thank you very much for the help!

**Reply by DubyaDude** [reply]
Great! Glad it was an easy fix. I _believe_ there's a way to ship .NET 8 installer with programs, I'll make an issue off of this and do some more research to see if we can make this a non-issue in future releases.
